File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);


    for (int i = 0; i < h; i++)
    {
        for (int k = 0; k < h; k++)
        {
            if (k < h - i - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: pset6/pset6/mario/mario.py
from cs50 import get_int


# def main function
def main():
    height = 0
    while (height < 1 or height > 8):  # make a loop with condition
        height = get_int("Height: ")

    for i in range(0, height):  # make a "array"
        for j in range(0, height):
            if j < height - i - 1:
                print(" ", end="")  # print without auto \n
            else:
                print("#", end="")  # print without auto \n
        print()  # print to new line array


main()